src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28711.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
```

Comment:
the test only tests for the app not crashing, not that the field corresponding to the x:Name is present (do you expect it to be ?), or that both XAML inflators are assigning it. Could you please move this test to Xaml.UnitTests, and test for both inflators ? thanks

Reply:
@StephaneDelcroix I’ve moved the UI test to UnitTests, similar to how it's done for GradientStop.